examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,132 @@
+"""
+Demonstrate basic loading and plotting of cortical surface data
+on the example of seed-based resting state functional connectivity.
+===================================================================
+The dataset that is a subset of the enhanced NKI Rockland sample
+(http://fcon_1000.projects.nitrc.org/indi/enhanced/, Nooner et al, 2012)
+
+Resting state fMRI scans (TR=645ms) of 102 subjects were preprocessed
+(https://github.com/fliem/nki_nilearn) and projected onto the Freesurfer
+fsaverage5 template (Dale et al, 1999, Fischl et al, 1999). For this example
+we use the time series of a single subject's left hemisphere.
+
+The Destrieux parcellation (Destrieux et al, 2010) in fsaverage5 space as
+distributed with Freesurfer is used to select a seed region in the posterior
+cingulate cortex.
+
+Functional connectivity of the seed region to all other cortical nodes in the
+same hemisphere is calculated using Pearson product-moment correlation
+coefficient.
+
+The :func:`nilearn.plotting.plot_surf_stat_map` function is used
+to plot the resulting statistical map on the (inflated) pial surface.
+
+See :ref:`plotting` for more details.
+
+References
+----------
+
+Nooner et al, (2012). The NKI-Rockland Sample: A model for accelerating the
+pace of discovery science in psychiatry. Frontiers in neuroscience 6, 152.
+URL http://dx.doi.org/10.3389/fnins.2012.00152
+
+Dale et al, (1999). Cortical surface-based analysis.I. Segmentation and
+surface reconstruction. Neuroimage 9.
+URL http://dx.doi.org/10.1006/nimg.1998.0395
+
+Fischl et al, (1999). Cortical surface-based analysis. II: Inflation,
+flattening, and a surface-based coordinate system. Neuroimage 9.
+http://dx.doi.org/10.1006/nimg.1998.0396
+
+Destrieux et al, (2010). Automatic parcellation of human cortical gyri and
+sulci using standard anatomical nomenclature. NeuroImage, 53, 1.
+URL http://dx.doi.org/10.1016/j.neuroimage.2010.06.010.
+"""
+
+###############################################################################
+from nilearn import plotting
+from nilearn import datasets
+from scipy import stats
+import nibabel as nb
+import numpy as np
+
+###############################################################################
+# Retrieve the data
+nki_dataset = datasets.fetch_nki_enhanced_surface(n_subjects=1)
+
+# Fsaverage5 left hemisphere surface mesh files
+fsaverage5_pial = nki_dataset['fsaverage5_pial_left'][0]
+fsaverage5_inflated = nki_dataset['fsaverage5_infl_left'][0]
+sulcal_depth_map = nki_dataset['fsaverage5_sulc_left'][0]
+
+# Destrieux parcellation left hemisphere in fsaverage5 space
+# TO DO: include in nki_dataset for fetching
+destrieux = '/home/julia/nilearn_testing/lh.aparc.a2009s.annot'
```

Comment:
Actually I am not sure it is a different parcellation (It is called 2009 within Freesurfer but the final atlas was published 2010), but anyhow the main difference is that this is surface space and the one in Nilearn is a volume, right?